python/doc/source/conf.py

```diff
@@ -6,11 +6,12 @@
 import os
 import sys
 
+import jupytext_process
```

Comment:
This is wrong. The reason the import of `jupytext_process` was on line 13 (original numbering) is that it is a file in the source directory, and therefore it needs lines 11